pkgs/build-support/appimage/default.nix

```diff
@@ -26,7 +26,7 @@ rec {
     ];
   };
 
-  extract = args@{ name ? "${args.pname}-${args.version}", postExtract ? "", src, ... }: pkgs.runCommand "${name}-extracted" {
+  extract = args@{ pname, version , postExtract ? "", src, ... }: pkgs.runCommand "${pname}-${version}-extracted" {
```

Comment:
is this fine to do? not sure

Reply:
Maybe we should abort if name is passed with an appropriate error?

Reply:
good idea. please look at it again.